packages/gatsby-plugin-google-tagmanager/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
Ugh, Kyle! Why did you include a babelrc there??

At some point we need to just centrally manage all the npmignore, gitignore, etc. files.

You mind removing both then?